src/utils/metadata.py

```diff
@@ -0,0 +1,43 @@
+import json
+import logging
+import os
+from hack_sound_server.configure import get_metadata_path
+from hack_sound_server.configure import get_sounds_dir
+
+
+_logger = logging.getLogger(__name__)
+
+
+def _read_in_metadata(metadata, user_type):
+    for sound_event_id in metadata:
+        metadata[sound_event_id]["sound-file"] =\
+            os.path.join(get_sounds_dir(user_type),
+                         metadata[sound_event_id]["sound-file"])
+
+
+def load_metadata(user_type):
+    metadata_path = get_metadata_path(user_type)
+
+    if os.path.exists(metadata_path):
+        with open(metadata_path, "r") as metadata_file:
+            # TODO
+            # Check valid JSON schema.
+            metadata = json.load(metadata_file)
+            _read_in_metadata(metadata, user_type)
+            return metadata
+    else:
+        msg = "The metadata file at '%s' does not exist." % metadata_path
+        _logger.warning(msg)
+        return None
+
+
+def read_and_parse_metadata():
+    system_metadata = load_metadata("system")
+    user_metadata = load_metadata("user")
+    if system_metadata is None:
```

Comment:
These checks still seem strange, but I think they are correct...

Unless maybe we should assume that the system metadata file is always there, and if it isn't, then it's a much bigger problem than missing the user metadata file?

Reply:
I also think that the system metadata file always exists, I just was handling this extreme case. Another option could be to do...
```
    if system_metadata is None:
        return None
```

Reply:
I would at least `_logger.error()` in that case as well